extensions/amp-accordion/1.0/amp-accordion.js

```diff
@@ -0,0 +1,150 @@
+/**
+ * Copyright 2020 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import * as Preact from '../../../src/preact';
+import {Accordion, AccordionSection} from './accordion';
+import {PreactBaseElement} from '../../../src/preact/base-element';
+import {childElementsByTag, toggleAttribute} from '../../../src/dom';
+import {devAssert, userAssert} from '../../../src/log';
+import {dict, memo} from '../../../src/utils/object';
+import {isExperimentOn} from '../../../src/experiments';
+import {toArray} from '../../../src/types';
+import {useLayoutEffect} from '../../../src/preact';
+
+/** @const {string} */
+const TAG = 'amp-accordion';
+
+const SECTION_SHIM_PROP = '__AMP_S_SHIM';
+const HEADER_SHIM_PROP = '__AMP_H_SHIM';
+
+class AmpAccordion extends PreactBaseElement {
+  /** @override */
+  init() {
+    const {element} = this;
+
+    const mu = new MutationObserver(() => {
+      this.mutateProps(getState(element, mu));
+    });
+    mu.observe(element, {
+      attributeFilter: ['expanded'],
```

Comment:
It's cool we won't need `[data-expand]` anymore: https://amp.dev/documentation/components/amp-accordion/#[data-expand]